src/Uno.UI.TestComparer/Uno.UI.TestComparer.csproj

```diff
@@ -2,14 +2,16 @@
 
 	<PropertyGroup>
 		<OutputType>Exe</OutputType>
-		<TargetFramework>net472</TargetFramework>
-	  <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
-  </PropertyGroup>
+		<TargetFramework>net6.0-windows</TargetFramework>
+		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+		<UseWPF>true</UseWPF>
+		<NoWarn>SYSLIB0014;CS1998</NoWarn>
+	</PropertyGroup>
 
 	<ItemGroup>
-	  <None Remove="Properties\launchSettings.json" />
-	</ItemGroup>	
+		<None Remove="Properties\launchSettings.json" />
+	</ItemGroup>
```

Comment:
I'm assuming that this was supposed to be here? Just double checking in case

Reply:
Yes, it's an indentation adjustment. Thanks :)